shared/src/iosMain/kotlin/co/touchlab/kampstarter/KoinIOS.kt

```diff
@@ -9,37 +9,50 @@ import com.squareup.sqldelight.db.SqlDriver
 import com.squareup.sqldelight.drivers.native.NativeSqliteDriver
 import kotlinx.cinterop.ObjCClass
 import kotlinx.cinterop.getOriginalKotlinClass
+import org.koin.core.Koin
+import org.koin.core.KoinApplication
+import org.koin.core.module.Module
 import org.koin.core.parameter.parametersOf
 import org.koin.core.qualifier.Qualifier
+import org.koin.core.qualifier.named
 import org.koin.dsl.module
 import platform.Foundation.NSUserDefaults
 
+val userDefaultsQualifier = named("userDefaults")
 actual val platformModule = module {
     single<Settings> {
-        val userDefaults = NSUserDefaults(suiteName = "KAMPSTARTER_SETTINGS")
-        AppleSettings(userDefaults)
+        // Note that Koin can't bind Objective-C types because they have no KClass. So we bind NSUserDefaults
+        //  as Any instead and ue use a Qualifier to specify what we're looking for.
+        AppleSettings(get<Any>(userDefaultsQualifier) as NSUserDefaults)
     }
     single<SqlDriver> { NativeSqliteDriver(KampstarterDb.Schema, "kampstarterdb") }
 
     val baseKermit = Kermit(NSLogLogger()).withTag("KampKit")
     factory { (tag: String?) -> if (tag != null) baseKermit.withTag(tag) else baseKermit }
 }
 
-object KoinIOS {
-    val koin = initKoin { }
+fun KoinApplication.module(moduleDeclaration: Module.() -> Unit) {
```

Comment:
Also added some other extension functions to make module declaration nicer in Swift, but there's probably still refining to do here.